kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/aggregation/settings/KoverParametersProcessor.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.aggregation.settings
+
+import kotlinx.kover.gradle.aggregation.settings.dsl.intern.KoverSettingsExtensionImpl
+import org.gradle.api.provider.HasMultipleValues
+import org.gradle.api.provider.ProviderFactory
+
+
+internal object KoverParametersProcessor {
+    fun process(settingsExtension: KoverSettingsExtensionImpl, providers: ProviderFactory) {
+        if (providers.gradleProperty("kover").isPresent) {
+            settingsExtension.coverageIsEnabled.set(true)
+        }
+
+        settingsExtension.reports.includedProjects.readAppendableArgument(providers, "kover.classes.from")
+        settingsExtension.reports.excludedProjects.readAppendableArgument(providers, "kover.classes.from.excludes")
+        settingsExtension.reports.excludedClasses.readAppendableArgument(providers, "kover.classes.excludes")
+        settingsExtension.reports.includedClasses.readAppendableArgument(providers, "kover.classes.includes")
+    }
+
+    private fun HasMultipleValues<String>.readAppendableArgument(providers: ProviderFactory, propertyName: String) {
+        val propertyProvider = providers.gradleProperty(propertyName)
+        if (propertyProvider.isPresent) {
+            val arg = propertyProvider.get().parseCollection()
+            if (!arg.append) {
+                empty()
+            }
+            println("Property '$propertyName' = ${arg.values}")
+            addAll(arg.values)
+        }
+    }
+
+    private fun String.parseCollection(): ArgCollection {
+        val append = startsWith('+')
```

Comment:
Just curious, is this notation widespread among Gradle plugins?

Reply:
No, but I do not know if there is a generally accepted syntax for such functionality.